src/playercards/cards/WellConnected.ttslua

```diff
@@ -0,0 +1,52 @@
+local playmatApi = require("playermat/PlaymatApi")
+
+local display = false
+local count = 0
+local modValue = 5
+local loopId = nil
+
+local b_display = {
+    index = 0,
+    click_function = 'none',
+    function_owner = self,
+    position = {-0.25,1,-0.75},
+    font_color = {1,1,1,100},
+    font_size = 250,
+    width = 0,
+    height = 0
```

Comment:
I think the button should get a background and be moved into the upper right corner. See here for inspiration: ![image](https://github.com/argonui/SCED/assets/97286811/6e2ea09c-ec58-40c2-9b5c-c3f45131f029)